src/ATen/native/quantized/QuantizedMaxPool2d.cpp

```diff
@@ -32,7 +33,7 @@ class QMaxPool_arr_args final {
       bool ceil_mode) {
     // Now we only support Byte, qint is not supported.
     TORCH_CHECK(
-        qx.scalar_type() == c10::ScalarType::Byte,
+        qx.scalar_type() == c10::ScalarType::Byte || qx.scalar_type() == c10::ScalarType::Char,
         "QuantizedMaxPool2d only supports Byte for xpu now");
```

Comment:
Make the error log correct.

Reply:
Has fixed, thanks for your kind reminding.